src/components/v2v3/V2V3Project/V2V3FundingCycleSection/modals/DistributePayoutsModal.tsx

```diff
@@ -151,7 +159,7 @@ export default function DistributePayoutsModal({
           >
             <FormattedNumberInput
               placeholder="0"
-              value={distributionAmount}
+              value={grossAvailableAmount}
```

Comment:
**issue** I havent tested, but just from reading the code, this looks like a potential issue. `grossAvailableAmount` isn't set by the user (or influenced by any mutable value); `distributionAmount` is (via `setDistributionAmount`).

Should we be changing this?

Reply:
You are right, I'm sorry. 

I guess that I've misread distributable with distributionAmount, things should be cool now.I've just added precision to max button and in starting hook.